src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs

```diff
@@ -239,7 +239,15 @@ public static class PublishedContentExtensions
             // Check if the culture has been set, otherwise use from Umbraco.
             if (culture == null)
             {
-                culture = UmbracoContext.Current.PublishedContentRequest.Culture;
+                if (UmbracoContext.Current.PublishedContentRequest != null)
```

Comment:
Sorry, I didn't mean as an extra if/else, but to add to the condition:

    if (UmbracoContext.Current != null && UmbracoContext.Current.PublishedContentRequest != null)

As if `UmbracoContext.Current` was null, then checking `UmbracoContext.Current.PublishedContentRequest` would throw the null reference.